lms/djangoapps/teams/views.py

```diff
@@ -322,14 +327,15 @@ def get(self, request):
             )
 
         queryset = CourseTeam.objects.filter(**result_filter)
+        # We will always use name as either a primary or secondary sort.
```

Comment:
Note that although @explorerleslie and I talked about this before she left, secondary sort by name is not consistent with what is in https://openedx.atlassian.net/browse/TNL-1937. I have asked Lou for clarification and will change this if necessary in my next PR, which will add sorting by last_activity_at to this endpoint.